server/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -148,6 +148,45 @@ public void testNestedLoop_TablesAreSwitchedIfLeftIsSmallerThanRight() {
         assertThat(((Reference) ((Collect) nl.left()).collectPhase().toCollect().get(0)).ident().tableIdent().name(), is("users"));
     }
 
+    @Test
+    public void test_nestedloop_tables_are_not_switched_when_stats_are_negative() {
+        txnCtx.sessionSettings().setHashJoinEnabled(false);
+        QueriedSelectRelation mss = e.analyze("select * from users, locations where users.id = locations.id");
```

Comment:
Could we just change the join comparison to not use the `=` operator to force a NestedLoop join? I think this would be more clear rather than to explicit disable hashjoins.

Reply:
I think the decision what join operation is used internally, should be decided by the database and not depending on SQL ?

Reply:
I updated the test, i hope it's ok now.